README.md

```diff
@@ -35,7 +35,7 @@ If you want to add a new image, use .png format if possible, and if it’s not a
 - Getting Started
 - Installation pages
 
-For translating text in the Jamulus application itself, please see this guide.?
+For translating text in the Jamulus application itself, please see [TRANSLATING.md](https://github.com/jamulussoftware/jamulus/blob/master/)
```

Comment:
Is that link _the_ link or just sort of getting there?

Reply:
Don't worry - I do the same with my real time text chat messages... :)  (Edit is a very useful tool.  Preview is overrated.)